src/main/java/no/unit/nva/database/UserDb.java

```diff
@@ -129,6 +133,36 @@ public void setUsername(String username) {
         this.username = username;
     }
 
+    @JacocoGenerated
+    @DynamoDBAttribute(attributeName = "givenName")
+    public String getGivenName() {
+        return givenName;
+    }
+
+    /**
+     * Method for using only for DynamoDb mapper. Do not use. Use the builder instead.
```

Comment:
Can you change the "Method for using only for DynamoDb mapper" to "Method to be used only by DynamoDb mapper"